update

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+// This script updates the index files list.js and list.txt in the bin directory,
+// as well as the soljson-latest.js files.
+
+fs.readdir(path.join(__dirname, '/bin'), function (err, files) {
+  if (err) return err
+
+  const parsedList = files
+    .map(function (file) { return file.match(/^soljson-v([0-9.]*)-.*.js$/) })
+    .filter(function (version) { return version })
+    .map(function (pars) { return { path: pars[0], version: pars[1] } })
+
+  const parsedObject = parsedList
+    .reduce(function (prev, next) {
+      prev[next.version] = next.path
+      return prev
+    }, {})
+
+  const pathList = parsedList
```

Comment:
Rename these to more meaningful variables:
- `builds` (all builds)
- `releases`